src/plugins/search/plugin.search.js

```diff
@@ -201,21 +197,14 @@ BookReader.prototype.search = function(term = '', overrides = {}) {
         ? options.success.call(this, searchInsideResults, options)
         : this.BRSearchCallback(searchInsideResults, options);
     }
-    cleanup();
-  };
-
-  const beforeSend = (xhr) => {
-    this.searchXHR = xhr;
-    window.BRSearchInProgress = processSearchResults;
   };
 
   this.trigger('SearchStarted', { term: this.searchTerm, instance: this });
   return $.ajax({
     url: url,
     dataType: 'jsonp',
     cache: true,
-    beforeSend,
-    jsonpCallback: 'BRSearchInProgress'
+    beforeSend: xhr => { this.searchXHR = xhr; },
```

Comment:
interesting, we don't need `jsopCallback` anymore?

Reply:
Testcafe was giving a weird error about `BRSearchInProgress` not being defined? So we're now using `$.ajax`'s implicit jsonp callback. Should behave the same! (Might cause caching issues now I think about it maybe?)